nautobot/docs/development/release-checklist.md

```diff
@@ -172,39 +172,19 @@ Build the images locally:
 ```no-highlight
 for ver in 3.6 3.7 3.8 3.9; do
   export INVOKE_NAUTOBOT_PYTHON_VER=$ver
-  invoke buildx --target final --tag networktocode/nautobot-py${INVOKE_NAUTOBOT_PYTHON_VER}:local
-  invoke buildx --target final-dev --tag networktocode/nautobot-dev-py${INVOKE_NAUTOBOT_PYTHON_VER}:local
+  invoke buildx --target final --tag networktocode/nautobot-py${INVOKE_NAUTOBOT_PYTHON_VER}:local &
+  invoke buildx --target final-dev --tag networktocode/nautobot-dev-py${INVOKE_NAUTOBOT_PYTHON_VER}:local &
```

Comment:
One downside of this change is that it won't be obvious if any of the builds fails (which I have had happen before due to Docker Desktop running out of disk space).

Reply:
Ah I've never had that. Outside of using a more advanced solution this is really just a stop-gap until we wire this up to CI.